From 01f714ee022ecb2667ca8ba909138b1af4cfff2c Mon Sep 17 00:00:00 2001
From: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Date: Wed, 18 Jan 2023 08:46:59 +0100
Subject: counter: fix dependency references for config MICROCHIP_TCB_CAPTURE

Commit dfeef15e73ca ("counter: microchip-tcp-capture: Add appropriate arch
deps for TCP driver") intends to add appropriate dependencies for the
config MICROCHIP_TCB_CAPTURE. It however prefixes the intended configs with
CONFIG, but in Kconfig files in contrast to source files, the configs are
referenced to without prefixing them with CONFIG.

Fix the dependency references due to this minor misconception.

Fixes: dfeef15e73ca ("counter: microchip-tcp-capture: Add appropriate arch deps for TCP driver")
Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Link: https://lore.kernel.org/r/20230118074659.5909-1-lukas.bulwahn@gmail.com/
Signed-off-by: William Breathitt Gray <william.gray@linaro.org>
---
 drivers/counter/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/counter/Kconfig b/drivers/counter/Kconfig
index 90602536fb13f..b5ba8fb02cf72 100644
--- a/drivers/counter/Kconfig
+++ b/drivers/counter/Kconfig
@@ -63,7 +63,7 @@ config INTERRUPT_CNT
 
 config MICROCHIP_TCB_CAPTURE
 	tristate "Microchip Timer Counter Capture driver"
-	depends on CONFIG_SOC_AT91SAM9 || CONFIG_SOC_SAM_V7 || COMPILE_TEST
+	depends on SOC_AT91SAM9 || SOC_SAM_V7 || COMPILE_TEST
 	depends on HAS_IOMEM && OF
 	select REGMAP_MMIO
 	help
-- 
cgit 

